#
# Removed warning message being displayed for non-existing exclude directories.
# See https://github.com/qarmin/czkawka/pull/1795
#
--- a/czkawka_core/src/common/directories.rs
+++ b/czkawka_core/src/common/directories.rs
@@ -79,7 +79,7 @@ impl Directories {
                 break;
             }
 
-            let (dir, msg) = Self::canonicalize_and_clear_path(&path, false);
+            let (dir, msg) = Self::canonicalize_and_clear_path(&path, is_excluded);
 
             messages.extend_with_another_messages(msg);
 
